### PR TITLE
README.md: reorg the content in README.md

### PR DESCRIPTION
**Pull request by zz-jason**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,127 +1,99 @@
...
-+ Stack Overflow: [https://stackoverflow.com/questions/tagged/tidb](https://stackoverflow.com/questions/tagged/tidb)
-
-+ Twitter: [https://twitter.com/PingCAP](https://twitter.com/PingCAP)
-
-+ Facebook: [https://www.facebook.com/pingcap2015](https://www.facebook.com/pingcap2015)
-
-
-+ Reddit: [https://www.reddit.com/r/TiDB](https://www.reddit.com/r/TiDB)
-
-## How Can I Help?
```
Comment (queenypingcap) [inline README.md]: I recommend keeping this part because this can give the audience a direction of how to engage when they first come.

**Comment by gingerkidney** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -1,127 +1,98 @@
...
 ## Acknowledgements
 
-Thank you to the [Kubernetes](https://github.com/kubernetes/community), [Apache](http://activemq.apache.org/becoming-a-committer.html) and [Docker](https://github.com/docker/community) community pages for providing us with inspiration.
-
-
-
+Thank you to the [Kubernetes](https://github.com/kubernetes/community),
+[Apache](http://activemq.apache.org/becoming-a-committer.html) and
+[Docker](https://github.com/docker/community) community pages for providing us
+with inspiration.
```
Comment (dcalvin) [inline README.md]:
```suggestion
with inspirations.
```

File: README.md [hunk 3]
```diff
@@ -1,127 +1,98 @@
...
+## Architecture
 
-You can reach out to the community members via any one of the following ways: 
+![TiDB Community Architecture](./architecture.svg)
 
-+ Slack: [https://pingcap.com/tidbslack/](https://pingcap.com/tidbslack/)
+## TiDB User Group
 
-+ Google Groups: [https://groups.google.com/forum/#!forum/tidb-user](https://groups.google.com/forum/#!forum/tidb-user)
+The TiDB User Groups are groups for facilitating communication and discovery of
```
Comment (dcalvin) [inline README.md]:
```suggestion
The TiDB User Groups (TUGs) are groups for facilitating communication and discovery of
```

File: README.md [hunk 4]
```diff
@@ -1,127 +1,98 @@
...
-+ Stack Overflow: [https://stackoverflow.com/questions/tagged/tidb](https://stackoverflow.com/questions/tagged/tidb)
+See [docs](https://pingcap.com/docs/) or
+[docs-cn](https://pingcap.com/docs-cn/) for user documents. Also you can get
+help in [AskTUG](https://asktug.com/) if you met any problem.
 
-+ Twitter: [https://twitter.com/PingCAP](https://twitter.com/PingCAP)
+## TiDB Developer Group
 
-+ Facebook: [https://www.facebook.com/pingcap2015](https://www.facebook.com/pingcap2015)
+### Communicating
```
Comment (dcalvin) [inline README.md]:
```suggestion
### Communication
```

File: README.md [hunk 5]
```diff
@@ -1,127 +1,98 @@
...
 
+The [communication](./communicating.md) page lists communication channels like
+chat, social medias, etc.
 
-+ Reddit: [https://www.reddit.com/r/TiDB](https://www.reddit.com/r/TiDB)
+For more specific topics, try to join
```
Comment (dcalvin) [inline README.md]:
```suggestion
For more specific topics, join
```

File: README.md [hunk 6]
```diff
@@ -1,127 +1,98 @@
...
 
-## How Can I Help?
+### Governance
 
-+ Try [TiDB](https://github.com/pingcap/tidb)! If you like it, star, share, and recommend it. 
+TiDB has the following types of groups that are officially supported:
 
-+ To dig deeper, read a design doc, e.g. [architecture](https://www.pingcap.com/blog/2017-05-23-perconalive17/), [How we build TiDB](https://www.pingcap.com/blog/2016-10-17-how-we-build-tidb/)
+* **Special Interest Groups (SIGs)** are persistent open groups that focus on a
+  part of the TiDB. SIGs must have open and transparent proceedings. Anyone is
```
Comment (dcalvin) [inline README.md]:
```suggestion
  module of TiDB. SIGs must have open and transparent proceedings. Anyone is
```

File: README.md [hunk 7]
```diff
@@ -1,127 +1,98 @@
...
+  welcome to participate and contribute provided they follow the TiDB [Code of
+  Conduct](./CODE_OF_CONDUCT.md). The purpose of a SIG is to own and develop a
+  part of TiDB.
```
Comment (dcalvin) [inline README.md]:
```suggestion
module of TiDB.
```

File: README.md [hunk 8]
```diff
@@ -1,127 +1,98 @@
...
 
-+ There's always a need for more test coverage. You can start with an area that has [low coverage](https://codecov.io/gh/pingcap/tidb).
+Contributions are welcomed and greatly appreciated. See
+[CONTRIBUTING.md](./CONTRIBUTING.md)
+for details on submitting patches and the contribution workflow.
 
-+ Give a talk at a local meetup or conference.
+You can start by finding an existing issue with the
+[help-wanted](https://github.com/pingcap/tidb/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
+label in the tidb repo. These issues are well suited for new contributors. BTW,
```
Comment (dcalvin) [inline README.md]:
```suggestion
label in the tidb repository. These issues are well suited for new contributors. BTW,
```

File: README.md [hunk 9]
```diff
@@ -1,127 +1,98 @@
...
+there's always a need for more test coverage. You can also start with an area
+that has [low coverage](https://codecov.io/gh/pingcap/tidb).
 
-## Learn to Use TiDB
+You can also join one of the [active working groups](./working-groups) to
+participate in a community activity. Start attending working group meetings and
```
Comment (dcalvin) [inline README.md]:
```suggestion
participate in a community activity. Start with attending working group meetings or
```

Comment (dcalvin) [reply README.md]: > Start attending working group meetings and

This sentence feels disconnected from the context. So I assume it's a suggestion on how to join working groups.

File: README.md [hunk 10]
```diff
@@ -1,127 +1,98 @@
...
+join the slack channel.
```
Comment (dcalvin) [inline README.md]:
```suggestion
joining the slack channel.
```

File: README.md [hunk 11]
```diff
@@ -1,127 +1,98 @@
...
+* [Become a Committer](./become-a-committer.md)
 
-+ TiDB Quick Start Guide
+## Learning Resources
 
-+ TiDB Operations Guide
+* [How we build TiDB](https://www.pingcap.com/blog/2016-10-17-how-we-build-tidb/)
+* [A Brief Introduction of TiDB](https://www.pingcap.com/blog/2017-05-23-perconalive17/)
+* [https://pingcap.com/docs](https://pingcap.com/docs)
+* [TiDB 源码阅读系列文章](https://pingcap.com/blog-cn/#TiDB-%E6%BA%90%E7%A0%81%E9%98%85%E8%AF%BB), in chinese
```
Comment (dcalvin) [inline README.md]: Suggest removing the Chinese links. Maybe some other articles in English, like TiDB Internal I II III? ...

File: README.md [hunk 12]
```diff
@@ -1,127 +1,98 @@
...
-Read the TiDB Weekly archives to catch up on all of the latest TiDB updates and hacks: [https://pingcap.com/weekly](https://pingcap.com/weekly)
-
-### Slideshare
-
-Another great resource for finding content! TiDB’s Slideshare account has slide decks from TiDB presentations -- you’re welcome to use and/or modify these slides when presenting to your own meetup group.
-
-### Tutorial Content
-
-If you're interested in writing a tutorial for TiDB, please contact us: [community@pingcap.com](mailto:community@pingcap.com)
+If you're interested in writing a tutorial for TiDB, please contact us:
```
Comment (dcalvin) [inline README.md]: This section feels strange - we don't offer anything tutorial, but instead, we ask for contribution....

File: communicating.md [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+
+The TiDB Blog is a great resource pool to find content! We post the latest TiDB
+news including new releases and technical articles there. Check out this page:
+[blog](https://pingcap.com/blog) or [blog-cn](https://pingcap.com/blog-cn)
+
+## TiDB Weekly
+
+[TiDB Weekly](https://pingcap.com/weekly) highlights content from the community
+along with the latest TiDB weekly update & news. You can read the TiDB Weekly
+archives to catch up on all of the latest TiDB updates and hacks.
```
Comment (dcalvin) [inline communicating.md]: > TiDB Weekly archives 

link?

File: communicating.md [hunk 14]
```diff
@@ -0,0 +1,31 @@
...
+
+[https://pingcap.com/tidbslack](https://pingcap.com/tidbslack)
+
+## Social Media
+
+### Blog
+
+The TiDB Blog is a great resource pool to find content! We post the latest TiDB
+news including new releases and technical articles there. Check out this page:
+[blog](https://pingcap.com/blog) or [blog-cn](https://pingcap.com/blog-cn)
```
Comment (dcalvin) [inline communicating.md]:
```suggestion
[blog](https://pingcap.com/blog) or [blog-cn](https://pingcap.com/blog-cn)(in Chinese).
```

File: README.md (hunk 12 above)
Comment (zz-jason) [reply README.md]: Yeah, I think we should remove it from this file.

File: README.md (hunk 11 above)
Comment (zz-jason) [reply README.md]: Have we translated the TiDB Internal articles?

File: communicating.md (hunk 13 above)
Comment (zz-jason) [reply communicating.md]: actually https://pingcap.com/weekly records both the current weekly report and all the archives. Is it ok to use the same link for `TiDB Weekly archives`? I think maybe we should specifies that readers can find all the archives on the same link?

Comment (dcalvin) [reply communicating.md]: Think it should be Ok if users can find them right in the same link

File: README.md (hunk 11 above)
Comment (dcalvin) [reply README.md]: We have. 
https://pingcap.com/blog/2017-07-11-tidbinternal1/
https://pingcap.com/blog/2017-07-11-tidbinternal2/
https://pingcap.com/blog/2017-07-11-tidbinternal3/

Comment (zz-jason) [reply README.md]: They're focused on the tidb architecture, while these Chinese articles are focused on the source code reading, they're different for the developers. I think both of them should be added to the learning resources.

Comment (zz-jason) [reply README.md]: BTW, we're going to use a big section to collect all the learning resources. It will be refactored in the future.

Comment (dcalvin) [reply README.md]: OK

File: communicating.md [hunk 15]
```diff
@@ -0,0 +1,31 @@
...
+Code of Conduct](./CODE_OF_CONDUCT.md). You can reach out to the community
+members via any one of the following ways:
+
+## Slack
+
+[https://pingcap.com/tidbslack](https://pingcap.com/tidbslack)
+
+## Social Media
+
+### Blog
```
Comment (dcalvin) [inline communicating.md]: Blog is strictly not social media...

**Pull request merged**